processes/reviewer-permissions-migration.md

```diff
@@ -0,0 +1,91 @@
+# reviewer-permissions-migration
+
+## Status
+
+implementable
+
+## Summary
+
+We should use the OWNERS files in each repository to manage the list
+of reviewers, instead of relying on the Github organization
+membership.
+
+## Motivation
+
+As the metal3 community expands, we are inevitably going to find that
+teams of people focus on different areas and different components. For
+example, we recently added the hardware-classification-controller,
+which is managed by some existing as well as new contributors. One of
+the things we have to balance as we add new contributors is the trust
+we extend and the obligations we place on them. I think we have grown
+to a point where we need to change how we do that.
+
+We have been using Github org membership as the way to indicate who
+has permission to use /lgtm as part of approving
+patches. Unfortunately, that extends to any repository in the org,
+which means we have to trust someone quite a lot before we invite them
+to be an org member. As we grow, this becomes more difficult to do
+with blanket permissions across all of our repositories.
+
+### Goals
+
+1. Transition from github org membership to OWNERS files for reviewer
+   permissions.
+
+### Non-Goals
+
+1. Define a new process for approving reviewers.
+2. Change the process for approving approvers.
+3. Change the permissions for managing the CI infrastructure.
+
```

Comment:
I am +1 with these non-goals as long as a process of adding the new reviewers is quick. I am only bit concern how we will give reviewers rights, /approve and /lgtm rights in the future. I totally agree that our growing projects needs some sort of gatekeeping, but how and when should we define the process for approving reviewers and promotion of members to have /lgtm for example?

Reply:
this was defined in #94

Reply:
Thanks @maelk everything was well defined on #94